src/systems/audio-debug-system.js

```diff
@@ -3,6 +3,7 @@ import audioDebugVert from "./audio-debug.vert";
 import audioDebugFrag from "./audio-debug.frag";
 import { DistanceModelType } from "../components/audio-params";
 import { getWebGLVersion } from "../utils/webgl";
+import { getMeshes } from "../components/teleporter";
```

Comment:
I'd suggest moving getMeshes out of teleporter.js and into a shared utility module, since it isn't actually specific to teleportation.

Reply:
Refactored to aframe-utils.js